config/services/dataplane_v1beta1_openstackdataplaneservice_libvirt.yaml

```diff
@@ -4,13 +4,4 @@ metadata:
   name: libvirt
 spec:
   label: dataplane-deployment-libvirt
-  role:
-    name: "Deploy EDPM libvirt"
-    hosts: "all"
-    strategy: "linear"
-    tasks:
-      - name: "libvirt"
-        import_role:
-          name: "osp.edpm.edpm_libvirt"
-        tags:
-          - "edpm_libvirt"
+  playbook: osp.edpm.libvirt
```

Comment:
why are these showing up in the diff? Have you rebased on master?

Reply:
Rebased again now and picked up two additional commits. Seems to have outdated this